### PR TITLE
ContentItemXSIAM default fromversion is now 8.0.0

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Related Issues
fixes: [link to issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-10304)

### CODE REVIEW

**Review by dantavori — APPROVED** [review]
need to fix tests

File: .changelog/4243.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Change the minimum version of ContentItemXSIAM to 8.0.0
```
Comment (ilaner) [inline .changelog/4243.yml:2]:
```suggestion
- description: Change the default `fromversion` of XSIAM content items to 8.0.0
```

**Comment by thefrieddan1** [comment]
[SDK nightly](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/1014310) passed successfully ( besides unrelated failure on slack notification )

File: demisto_sdk/commands/upload/tests/uploader_test.py [hunk 2]
```diff
@@ -897,10 +897,8 @@ def test_upload_without_skip_validate(self, mocker, path: Path, version: str):
                     "metadata.json",
                     "pack_metadata.json",
                     "XSIAMDashboards/",
-                    "XSIAMDashboards/xsiamdashboard-MyDashboard.json",
                     "XSIAMDashboards/external-xsiamdashboard-MyDashboard.json",
                     "Triggers/",
-                    "Triggers/trigger-MyTrigger.json",
```
Comment (dantavori) [inline demisto_sdk/commands/upload/tests/uploader_test.py:903]: why are these removed?

**Review by dantavori — APPROVED** [review]
nice, see note

File: demisto_sdk/commands/upload/tests/uploader_test.py (hunk 2 above)
Comment (thefrieddan1) [reply demisto_sdk/commands/upload/tests/uploader_test.py:903]: Setting the default fromversion of this items to 8.0.0 prevents uploads only the external version of the files.
